cg/constants/delivery.py

```diff
@@ -116,8 +117,8 @@
 MIP_RNA_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
     {"fusion", "star-fusion"},
     {"fusion", "arriba"},
-    {"cram"},
-    {"cram-index"},
+    {AlignmentFileTag.CRAM},
+    {AlignmentFileTag.CRAM_INDEX},
     {"fusion", "vcf"},
     {"fusion", "vcf-index"},
     {"salmon-quant"},
```

Comment:
Would be nice to update these with the new tags as well.

Reply:
done